xbedump.c

```diff
@@ -438,6 +438,19 @@ if (option_flag & 0x00000002) {
        
          sechdr = (XBE_SECTION *)(((char *)xbe) + (int)header->Sections - (int)header->BaseAddress);
          for (i = 0; i < header->NumSections; i++, sechdr++) {
+ if (option_flag & 0x00000100) {
+    char *name = ((char  *)xbe)+(int)sechdr->SectionName-(int)header->BaseAddress;
+    if (!strcmp(name,".text")) {
+        printf("0x%08X ", sechdr->VirtualAddress);
+    }else if (!strcmp(name,".bss") || !strcmp(name,".data") || !strcmp(name,".rdata")) {
```

Comment:
Would there ever be any reason why someone wouldn't want to map the rest of the sections?